src/annotator/anchoring/pdf.ts

```diff
@@ -680,7 +674,7 @@ export async function describe(
     end: pageOffset + endPos.offset,
   } as TextPositionSelector;
 
-  const quote = TextQuoteAnchor.fromRange(root, textRange).toSelector();
+  const quote = TextQuoteAnchor.fromRange(pageView.div, textRange).toSelector();
```

Comment:
The first argument here can be any ancestor element that contains the entire text range. The output is unaffected by which one is chosen.